Dockerfile

```diff
@@ -0,0 +1,34 @@
+# FOR ARM64, 
+FROM --platform=linux/amd64 python:3.9-slim-buster
+
+# Install system dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+       ffmpeg \
+       libsm6 \
+       libxext6 \
+       binutils \
+       libproj-dev \
+       gdal-bin \
+       libgdal-dev \
+       curl
```

Comment:
This smells funny.  What is the justification for each of these?  Are they actually being used?

Reply:
slimmed this down, includid python3-pip and only stuff needed for rasterio